runtime/instance-cache.cpp

```diff
@@ -638,9 +638,6 @@ class InstanceCache {
   }
 
   void fire_warning(const InstanceDeepCopyVisitor &detach_processor, const char *class_name) noexcept {
-    if (detach_processor.is_depth_limit_exceeded()) {
-      php_warning("Depth limit exceeded on cloning instance of class '%s' into cache", class_name);
-    }
```

Comment:
Why was it deleted? Was it only for cycle refs?

Reply:
Yep, the main users of this feature were cycles. 
Now, when cycles are supported, the memory limit is enough as one limit for everything.